ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -163,10 +165,20 @@ export class WorkflowDetails extends React.Component<RouteComponentProps<any>, W
                                     </div>
                                 ))}
                         </div>
+                        <div>
+                            <input type='checkbox' id='followLogStream' onClick={() => this.setState({followLogStream: !this.state.followLogStream})} />
+                            <label htmlFor='followLogStream'>Follow log stream?</label>
```

Comment:
```suggestion
                            <label htmlFor='followLogStream'>Stream Logs</label>
```
Is a better name for this

Reply:
Also, this doesn't seem to be in the correct place at all, see: 
![image](https://user-images.githubusercontent.com/614838/83810595-3fc8ed00-a66d-11ea-9bdb-634312d7ab31.png)

Can you add this within the `WorkflowLogsViewer` component?